src/components/BulkEnrollmentPage/BulkEnrollment.scss

```diff
@@ -39,7 +43,12 @@
 }
 
 .sticky-stepper-footer {
```

Comment:
nit: I feel like this class is now not particularly well named? Maybe `sticky-footer-wrapper` would be better? Up to you.